Jenkinsfile-rf-pipeline

```diff
@@ -0,0 +1,308 @@
+                    }
+                    dir('cciskel') {
+                        git 'https://github.com/cgwalters/centos-ci-skeleton'
+                    }
+                    dir('sig-atomic-buildscripts') {
+                        git 'https://github.com/CentOS/sig-atomic-buildscripts'
+                    }
+
+                    // Set stage specific vars
+                    pipelineUtils.setStageEnvVars(currentStage)
+
+                    //Set our message topic, properties, and content
+                    (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("package.running")
+
+                    // Send message org.centos.prod.ci.pipeline.package.running on fedmsg
+                    sendMessage(messageProperties, messageContent)
+
+                    // Provision of resources
+                    pipelineUtils.provisionResources(currentStage)
+
+                    // Stage resources - RPM build system
+                    pipelineUtils.setupStage(current_stage, 'fedora-atomic-key')
+
+                    // Rsync Data
+                    pipelineUtils.rsyncData(currentStage)
+
+                    def package_props = "${ORIGIN_WORKSPACE}/logs/package_props.txt"
+                    def package_props_groovy = "${ORIGIN_WORKSPACE}/package_props.groovy"
+                    pipelineUtils.convertProps(package_props, package_props_groovy)
+                    load(package_props_groovy)
+
+                    // Teardown resources
+                    pipelineUtils.teardownResources(currentStage)
+
+                    // Set our message topic, properties, and content
+                    (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("package.complete")
+
+                    // Send message org.centos.prod.ci.pipeline.package.complete on fedmsg
+                    sendMessage(messageProperties, messageContent)
+
+                    pipelineUtils.checkLastImage(currentStage, checkRsyncDataDir=false)
+
+                }
+
+                currentStage = "ci-pipeline-ostree-compose"
+                stage(currentStage) {
+                    // Set stage specific vars
+                    pipelineUtils.setStageEnvVars(currentStage)
+
+                    //Set our message topic, properties, and content
+                    (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("compose.running")
+
+                    // Send message org.centos.prod.ci.pipeline.compose.running on fedmsg
+                    sendMessage(messageProperties, messageContent)
+
+                    // Provision resources
+                    pipelineUtils.provisionResources(currentStage)
+
+                    // Stage resources - ostree compose
+                    pipelineUtils.setupStage(current_stage, 'fedora-atomic-key')
+
+                    // Rsync Data
+                    pipelineUtils.rsyncData(currentStage)
```

Comment:
You changed the name of the method so this needs to be updated

Reply:
rsyncData() didn't change name. The method rsyncResults was renamed to runTaskAndReturnLogs(). 
This wouldn't need to be updated.

Reply:
Then why do we need runTaskAndReturnLogs() for?

Reply:
kk